### ISSUE TITLE
Ex Parte Report - PREP - Case Selection - determine if person is counted

### ISSUE THREAD

**Issue by C-Love** [Hennepin-County/MAXIS-scripts#1397]
We need to solidify the functionality for selecting cases for Ex Parte. 

Currently we are excluding cases with certain income on the case, even if that income is for a person that does not deem to the ABD HC eligibility. This is partially due to the cases being a little messy at this point and it is hard to get all the needed information from the ELIG list, this may improve as the unwinding completes, but we need to review. 

In the mean time, we want to try to include enrollees where there is income on the case that does not deem. 

We also need to consider spousal allocation and excluding these, maybe? Unclear at this time. 

Basically, cases with income and multiple people are complicated and we need to solve for our discrepancies.

**Comment by C-Love** [comment]
For now, we are going to ignore income when making an ex parte determination unless the person is enrolled in HC.
